Description: Add octal escape sequence character EOF of decimal value 255
Author: github.com/Simmesson
 TODO: find more info on the patch author.
Last-Update: 2016-11-27

--- hexinject-1.5.orig/argparser.h
+++ hexinject-1.5/argparser.h
@@ -148,10 +148,15 @@ int parseopt(int argc, char **argv)
             case 'h': //help
                 usage(stdout, NULL);
 
+            case '\377':
+                break;
+            
             case '?':
             default:
                 usage(stderr, NULL);
         }
+        if(ch == '\377') // Octal escape character sequence EOF. Decimal value 255.
+            break;
     }
     
     // check mode
